.github/release-please.yml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+path: anthos-bm-gcp-terraform
+changelogPath: CHANGELOG.md
```

Comment:
- I want the CHANGELOG file inside the `anthos-bm-gcp-terraform` folder to be updated

Should the path be the full path `anthos-bm-gcp-terraform/CHANGELOG.md`?
Or since we have the root folder defined with the `path` attribute can it just be the name of the changelog file?

Reply:
I think this is correct, the `changelogPath` should be relative to the `path` option.